game_of_thrones/app/views/houses/index.html.erb

```diff
@@ -0,0 +1,9 @@
+<h1>Welcome to the Game of Thrones</h1>
+<% @houses.each do |house| %>
+<div class="house">
+  <h3><%= link_to house.house_name, house_path(house) %></h3>
+</div>
+<% end %>
+
+<h2><a href="/houses/new">Create new House</a></h2>
+<h2><a href="/houses/2/edit">Update House</a></h2>
```

Comment:
see comment above. your views should not have hardcoded values like the one above